Doppler.Currency/Dtos/CurrencyDto.cs

```diff
@@ -12,5 +12,6 @@ public class CurrencyDto
         public decimal? BuyValue { get; set; }
         public string CurrencyName { get; set; }
         public string CurrencyCode { get; set; }
+        public bool NoPrice => SaleValue == 0 || BuyValue == 0;
```

Comment:
Is using both `SaleValue` and `BuyValue` really necessary?

Reply:
Really for me is not possible the scenario that one value is 0 and another no. Equaly with the changes adove the condition will be change and olnly ckeck with SaleValue